libs/core/src/main/java/com/sourcefuse/jarc/core/constants/CommonConstants.java

```diff
@@ -0,0 +1,23 @@
+package com.sourcefuse.jarc.core.constants;
+
+public final class CommonConstants {
+
+  public static final String NO_GRP_PRESENT =
+    "No group is present against given value";
+
+  public static final String FIND = "FIND";
```

Comment:
Can we remove unwanted constants for filter as that is going to be covered separately by harshad

Reply:
sure